src/App.tsx

```diff
@@ -90,15 +50,46 @@ function AppContent() {
         <TopBarContent setShowNavBar={setShowNavBar} />
       </TopBar>
 
-      <div className={classes.container}>
+      <Box
+        sx={{
```

Comment:
useMemo for these styles? It is a root level component and maybe worth that optimization.